.gitignore

```diff
@@ -1 +1,2 @@
 *.pyc
+tmp*
```

Comment:
Does this change actually need to be committed to the main repository? You can add files in `.git/info/exclude` for ones that are specific to you personally.